docs/DeveloperGuide.md

```diff
@@ -739,101 +638,80 @@ otherwise)
 
 ****
 
+**Use case: Delete question**
 
-**Use case: Mark question**
-
-1. User requests to list questions
-2. SETA shows list of questions
-3. User requests to mark a specific question as important
-4. SETA marks the question as important
+1. User requests to delete a specific question in the list.
+2. SETA deletes the question.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 2a. The given index is invalid.
 
-    * 3a1. SETA shows an error message.
+    * 2a1. SETA shows an error message.
 
-      Use case resumes at step 2.
+      Use case resumes at step 1.
 
 ****
 
+**Use case: Mark question**
 
-**Use case: Unmark question**
-
-1. User requests to list questions.
-2. SETA shows list of questions.
-3. User requests to mark a specific question as unimportant.
-4. SETA marks the question as unimportant.
+1. SETA shows list of questions.
```

Comment:
Since "User requests to list question" is not here anymore, I am not sure whether we need this step. It might be better to start with the step below ("User requests to mark a specific question in the list as important"). 

Would appreciate if we can get inputs from other members!